src/components/howTos.stories.mdx

```diff
@@ -33,3 +33,62 @@ const App = () => {
   );
 };
 ```
+
+## Implement view-only/observer/spectator experience
+
+With the release of version 2.6.0, React Component SDK now supports user-defined device permissions controls.
+
+In this version, we added optional parameter `deviceLabels: DeviceLabels | DeviceLabelTrigger` in `meetingManager.join()`. You could pass a `deviceLabels` of type `DeviceLabels` to select the devices from which the browser requests permission when joining the meeting. You could also pass a `deviceLabels` of `DeviceLabelTrigger` type, to set your customized `deviceLabelTrigger` which is triggered to request the device permissions. We also added a new function `invokeDeviceProvider(deviceLabels: DeviceLabels)`. This function can setup the `deviceLabelTrigger` based on the `deviceLabels` passed in, and trigger the `deviceLabelTrigger` to invoke the device permission.
+
+```tsx
+enum DeviceLabels {
```

Comment:
may need to replace `enum` with `const`

Reply:
I'm not following you. `DeviceLabels` is an `enum`, I think it's fine here. Let me know if you have any specific reason.

Reply:
Since `enum` is a data type that supported in TS, not JS.

Reply:
Oh I get it. I will change the language here to ts. 
Since we defined `DeviceLabels` in the React SDK, which is written in TS. I think keeping it as `enum` can make it simple and clear for our builder. When they write their own code, they could just follow the example here.

Reply:
I asked Kyu about the language in the doc before. His own word:
**In our documentation, we wrote examples in a pure JavaScript. Basically, the example code should work in the browser’s console.**

Reply:
Refer to other `enum` examples and change the language to `javascript`.